.pre-commit-hooks.yaml

```
@@ -0,0 +1,9 @@
+- id: mkdocs-build
+  name: mkdocs build
+  description: tests if your documentation builds successfully.
+  language: python
+  entry: mkdocs build
+  require_serial: true
+  pass_filenames: false
+  types_or: [markdown, yaml]
+  files: ((^|/)mkdocs\.ya?ml$)|(^docs/)
```

`types_or` was added in the version of pre-commit 2.9.0, so you must define the [`minimum_pre_commit_version`](https://pre-commit.com/#hooks-minimum_pre_commit_version):

```suggestion
  files: ((^|/)mkdocs\.ya?ml$)|(^docs/)
  minimum_pre_commit_version: 2.9.0
```